### ISSUE TITLE
Structure examples

### ISSUE THREAD

**Issue by kaaveland** [kaaveland/eugene#60]
I would like to set up examples of every lint and hint, and show snapshots of both json and markdown, as well as showing the trace and lint of the "workaround", where those exist.

**Comment by kaaveland** [comment]
I think I'd like to take a stab at generating a static site for these, using hugo to provide some layout and navigation of the eugene demo.

**Comment by kaaveland** [comment]
We have the structure we need after #61 and I'm making a new issue for generating a pages setup or something.
